src/RealtimeEffectPanel.cpp

```diff
@@ -1000,6 +1001,37 @@ RealtimeEffectPanel::RealtimeEffectPanel(
             break;
          }
    });
+
+   mUndoSubscription = UndoManager::Get(mProject).Subscribe(
+      [this](UndoRedoMessage message)
+      {
+         auto& trackList = TrackList::Get(mProject);
+
+         // Realtime effect UI is only updated on Undo or Redo
+         if (message.type == UndoRedoMessage::Type::UndoOrRedo)
```

Comment:
Also for the `Reset`?  You can make that happe with the History dialog

Reply:
Will do